issue16161.py

```diff
@@ -0,0 +1,44 @@
+from sympy import *

```

Comment:
Please import only the required functionalities for better efficiency of the code.

Reply:
Yes, I know this. I have seen the module.